### PR TITLE
Get rid of CocoaPods, switch FMDB to SPM, enable Swift standard libraries #changed

### PR DESCRIPTION
**Pull request by Kaspik**
## Changes:
- Get rid of CocoaPods
- Switch FMDB to SPM
- Enable Swift standard libraries
- Fix couple of warnings

## Closes following issues:
- Might `fix` https://github.com/Sequel-Ace/Sequel-Ace/issues/777

## Tested:
- Processors:
  - [ ] Intel
  - [x] Apple Silicon
- macOS Versions:
  - [ ] 10.12.x (Sierra)
  - [ ] 10.13.x (High Sierra)
  - [ ] 10.14.x (Mojave)
  - [ ] 10.15.x (Catalina)
  - [x] 11.x (Big Sur)
- Xcode Version:
- Localizations:
  - [ ] English
  - [ ] Spanish
  - [ ] Other (please specify)
## Additional notes:
Without CocoaPods our life might be way easier.

### CODE REVIEW

File: Frameworks/SPMySQLFramework/SPMySQLFramework.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -615,6 +615,7 @@
 			hasScannedForEncodings = 1;
 			knownRegions = (
 				en,
+				Base,
```
Comment (Jason-Morcos) [inline Frameworks/SPMySQLFramework/SPMySQLFramework.xcodeproj/project.pbxproj:618]: Intentional?

File: Podfile [hunk 2]
```diff
@@ -1,113 +0,0 @@
-source 'https://cdn.cocoapods.org/'
```
Comment (Jason-Morcos) [inline Podfile:1]: Should update the Sequel Ace readme to drop the `bundle exec pod install --repo-update` line! :)

File: sequel-ace.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -3733,6 +3709,7 @@
 		96BC47C72500A84F003C8105 /* Beta */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
+				ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = YES;
```
Comment (Jason-Morcos) [inline sequel-ace.xcodeproj/project.pbxproj:3712]: Looks like the flag got set to YES for beta and Distribution but not for all of the other configs? Looks like release is set to NO?

File: Frameworks/SPMySQLFramework/SPMySQLFramework.xcodeproj/project.pbxproj (hunk 1 above)
Comment (Kaspik) [reply Frameworks/SPMySQLFramework/SPMySQLFramework.xcodeproj/project.pbxproj:618]: Yes! Just to fix warning, not actually using it

File: Podfile (hunk 2 above)
Comment (Kaspik) [reply Podfile:1]: Riiight!

File: sequel-ace.xcodeproj/project.pbxproj (hunk 3 above)
Comment (Kaspik) [reply sequel-ace.xcodeproj/project.pbxproj:3712]: I believe not. It was set for the whole project, but excluded two targets (xib and sptunnel) - they were throwing warnings

**Review by Jason-Morcos — APPROVED** [review]

**Pull request merged**